aspnetcore/signalr/version-differences.md

```diff
@@ -96,21 +106,21 @@ connection.on("ReceiveMessage", (user, message) => {
 });
 ```
 
-After creating the client method, start the hub connection. Chain a `catch` method to log or handle errors.
+After creating the client method, start the hub connection. Chain a [catch](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Promise/catch) method to log or handle errors.
```

Comment:
This is the resource we link to in the JavaScript docs for core types like Promise, it's the Mozilla Developer Network docs. Not sure if we want to add this kind of link, it was just `code fenced` before. I can go back to that, but I like linking to as many API docs as we can :)